### PR TITLE
helm: Add support for passing arbitrary secrets

### PR DESCRIPTION
**Pull request by ekarlso**
**Checklist**
- [ X] Documentation added
- [ ] Tests updated

### CODE REVIEW

**Comment by Starefossen** [comment]
Nice work @ekarlso 👍

File: production/helm/loki/templates/statefulset.yaml [hunk 1]
```diff
@@ -51,6 +51,10 @@ spec:
             - "-{{ $key }}={{ $value }}"
           {{- end }}
           volumeMounts:
+{{ range .Values.secrets }}
+            - name: {{ . }}-secret
+              mountPath: /etc/loki/secrets/{{ . }}
+{{- end }}
```
Comment (Starefossen) [inline production/helm/loki/templates/statefulset.yaml]: Indent this to align with line 53?

File: production/helm/loki/templates/statefulset.yaml [hunk 2]
```diff
@@ -87,6 +91,11 @@ spec:
         - name: config
           secret:
             secretName: {{ template "loki.fullname" . }}
+{{- range .Values.secrets }}
+        - name: {{ . }}-secret
+          secret:
+            secretName: {{ . }}
+{{- end }}
```
Comment (Starefossen) [inline production/helm/loki/templates/statefulset.yaml]: Indent this to allign with line 91?

File: production/helm/loki/templates/statefulset.yaml [hunk 3]
```diff
@@ -51,6 +51,9 @@ spec:
             - "-{{ $key }}={{ $value }}"
           {{- end }}
           volumeMounts:
+{{- if .Values.extraVolumeMounts }}
```
Comment (steven-sheehy) [inline production/helm/loki/templates/statefulset.yaml]: Indent to be same level as config volume mount using `nindent` to match how everything else is indented in the file.

**Comment by ekarlso** [comment]
@steven-sheehy Any updates ? :)

**Pull request merged**